server/datastore/mysql/app_configs_test.go

```diff
@@ -114,7 +114,7 @@ func testAppConfigOrgInfo(t *testing.T, ds *Datastore) {
 
 	verify, err = ds.UserByEmail(context.Background(), email)
 	assert.Nil(t, err)
-	assert.False(t, verify.SSOEnabled)
+	assert.True(t, verify.SSOEnabled) // SSO stays enabled for user even when globally disabled
 }
```

Comment:
nit: if time allows we could enhance the integration tests to log an user even after SSO has been disabled.

Reply:
I don't follow. If admin globally disables SSO, an sso-enabled user won't be able to log in until admin sets their password.